fortls/objects.py

```diff
@@ -160,6 +181,22 @@ def check_scope(
                 return child
         return None
 
+    def check_import_scope(scope: Scope, var_name_lower: str):
+        for use_stmnt in scope.use:
+            if not type(use_stmnt) is Import:
+                continue
+            if use_stmnt.import_type == ImportTypes.ONLY:
+                # Check if name is in only list
+                if var_name_lower in use_stmnt.only_list:
+                    return ImportTypes.ONLY
+            # Get Get the parent scope
+            elif use_stmnt.import_type == ImportTypes.ALL:
+                return ImportTypes.ALL
+            # Skip looking for parent scope
+            elif use_stmnt.import_type == ImportTypes.NONE:
+                return ImportTypes.NONE
+        return None
+
```

Comment:
Note to self, this is only used in `AST.check_definitions` where the `interface` argument is set to `True`. Potentially it could have other usecases, but at the time it is being unused.